src/keycloak/keycloak_admin.py

```diff
@@ -2810,3 +2813,110 @@ def create_client_authz_client_policy(self, payload, client_id):
             data=json.dumps(payload),
         )
         return raise_error_from_response(data_raw, KeycloakPostError, expected_codes=[201])
+
+    def get_composite_client_roles_of_group(self, client_id, group_id):
+        """Get the composite client roles of the given group for the given client.
+
+        :param client_id: id of the client.
+        :type client_id: str
+        :param group_id: id of the group.
+        :type group_id: str
+        :return: the composite client roles of the group (list of RoleRepresentation).
+        :rtype: list
+        """
+        params_path = {"realm-name": self.realm_name, "id": group_id, "client-id": client_id}
+        data_raw = self.raw_get(
+            urls_patterns.URL_ADMIN_GROUPS_CLIENT_ROLES_COMPOSITE.format(**params_path)
+        )
+        return raise_error_from_response(data_raw, KeycloakGetError)
+
+    def get_role_client_level_children(self, client_id, role_id):
+        """Get the child roles of which the given composite client role is composed of.
+
+        :param client_id: id of the client.
+        :type client_id: str
+        :param role_id: id of the role.
+        :type role_id: str
+        :return: the child roles (list of RoleRepresentation).
+        :rtype: list
+        """
+        params_path = {"realm-name": self.realm_name, "role-id": role_id, "client-id": client_id}
+        data_raw = self.raw_get(urls_patterns.URL_ADMIN_CLIENT_ROLE_CHILDREN.format(**params_path))
+        return raise_error_from_response(data_raw, KeycloakGetError)
+
+    def get_user_credentials(self, user_id):
```

Comment:
I'm pretty sure this method already exists: https://github.com/marcospereirampj/python-keycloak/blob/master/src/keycloak/keycloak_admin.py#L638

Reply:
Ok I see, anyhow I think it would a good idea to rename it from `get_credentials` to `get_user_credentials`, along with `delete_credentials` to `delete_user_credentials`

Reply:
Maybe, but that'd be a breaking change, there might be users out there having scripts and applications expecting the method to be called `get_credentials`. If we want to go and force them to change their application/script code, I'd need some good arguments for why we have to change this :)